Package.swift

```diff
@@ -43,20 +43,20 @@ let package = Package(
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
-        .target(
-            name: "appstoreconnect-cli",
-            dependencies: ["AppStoreConnectCLI"]),
+        .target(name: "appstoreconnect-cli", dependencies: ["AppStoreConnectCLI"]),
+        .target(name: "Model"),
```

Comment:
Do we want to have a more implicit name here? like AppleStoreConnectCLI-Model?

Reply:
I don't think so, these modules are only visible to our package as far as I understand

Reply:
Also this package isn't built to be consumed, only to be compiled into the `asc` binary